samples/boards/nrf52/mesh/onoff_level_app/src/mesh/doer.c

```diff
@@ -0,0 +1,182 @@
+#include "common.h"
+
+void doer(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf, uint16_t opcode)
+{
+	uint8_t tid, tmp8;
+	int16_t tmp16;
+	int err = 0, tmp32;
+
+	struct net_buf_simple *msg;
+
+	struct server_state *state_ptr = model->user_data;
```

Comment:
We don't generally put type information in variable names, so please remove the `_ptr` part.